to_do_list/src/Components/SimpleDialog.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "../Style/SimpleDialog.css";
+
+function SimpleDialog(props) {
+  const { onClose, open, title, action } = props;
+
+  const handleClose = shouldUpdate => {
+    onClose(shouldUpdate, selectedValue);
+  };
+
+  let selectedValue = "";
+  return (
+    <dialog className="dialog" open={open} onClose={handleClose}>
+      <div className="dialogBox">
+        <h1 className="dialogTitle"> {title} </h1>
```

Comment:
using `<h1>` is useless here (and its not the same size as the h1 in the header if you noticed)
just use `<div>` or `<span>` and add your css to it...